src/main/kotlin/com/example/pokemon_api_final/enums/AllPokemons.kt

```diff
@@ -0,0 +1,530 @@
+package com.example.pokemon_api_final.enums
+
+enum class AllPokemons(
```

Comment:
Keeping them in a list was fine, that's an overkill for an ENUM.
Keeping their type in an enum is ok, the whole list, not really.

Please undo.